<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QDialog" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/image1/img/backgorund.jfif</normaloff>:/image1/img/backgorund.jfif</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/gamescreen/gamescreenimg/background.png);</string>
  </property>
  <widget class="QListWidget" name="projectListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>241</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/widgetbackground.png);</string>
   </property>
  </widget>
  <widget class="QListWidget" name="developerListWidget">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>40</y>
     <width>241</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/widgetbackground.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearmoneylabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Text</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:12px;
border-image: url(:/gamescreen/gamescreenimg/turuncu.png);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#7a4933;&quot;&gt;Year /Money&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buynowbutton">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>160</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border-image: url(:/gamescreen/gamescreenimg/buynowbutton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="buyprojectwidget">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>70</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/buywidgetbackground.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="dev1label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>560</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/devlabelbackground.jpeg);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#a66948;&quot;&gt;NAME SURNAME&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#a66948;&quot;&gt;speciality&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buydeveloper1button">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>640</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
border-image: url(:/gamescreen/gamescreenimg/hire3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="gamenextbutton">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>580</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
border-image: url(:/gamescreen/gamescreenimg/nextyearbutton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="gamebackbutton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/h.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="projectLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/labelbackground.png);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#7a4933;&quot;&gt;PROJECTS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="developerLabel">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>0</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/labelbackground.png);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#7a4933;&quot;&gt;DEVELOPERS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="buyprojectlabel">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>30</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/buylabelbackground.png);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; font-style:italic; color:#7a4933;&quot;&gt;Buy a project&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="dev2label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>560</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/devlabelbackground.jpeg);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#a66948;&quot;&gt;NAME SURNAME&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#a66948;&quot;&gt;speciality&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="dev5label">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>560</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/devlabelbackground.jpeg);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#a66948;&quot;&gt;NAME SURNAME&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#a66948;&quot;&gt;speciality&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="dev3label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>560</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/devlabelbackground.jpeg);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#a66948;&quot;&gt;NAME SURNAME&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#a66948;&quot;&gt;speciality&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="dev4label">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>560</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/devlabelbackground.jpeg);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#a66948;&quot;&gt;NAME SURNAME&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#a66948;&quot;&gt;speciality&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buydeveloper2button">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>640</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
border-image: url(:/gamescreen/gamescreenimg/hire3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buydeveloper3button">
   <property name="geometry">
    <rect>
     <x>545</x>
     <y>640</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
border-image: url(:/gamescreen/gamescreenimg/hire3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buydeveloper4button">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>640</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
border-image: url(:/gamescreen/gamescreenimg/hire3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buydeveloper5button">
   <property name="geometry">
    <rect>
     <x>865</x>
     <y>640</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
border-image: url(:/gamescreen/gamescreenimg/hire3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteprojebutton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
border-image: url(:/gamescreen/gamescreenimg/binbutton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="deletedeveloperbutton">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>350</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
border-image: url(:/gamescreen/gamescreenimg/binbutton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="buydevlabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
border-image: url(:/gamescreen/gamescreenimg/devlabelbackground.jpeg);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#a66948;&quot;&gt;HIRE A DEVELOPER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="namelabelkemal">
   <property name="geometry">
    <rect>
     <x>797</x>
     <y>30</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Text</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
border-image: url(:/gamescreen/gamescreenimg/turuncu.png);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#7a4933;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/whitebackground.jpeg);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>410</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/whitebackground.jpeg);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>410</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/whitebackground.jpeg);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>410</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/whitebackground.jpeg);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>410</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/whitebackground.jpeg);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="selectedProjectLabel">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>240</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:8px;
border-image: url(:/gamescreen/gamescreenimg/morlabel.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="selectedDeveloperLabel">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>240</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:8px;
border-image: url(:/gamescreen/gamescreenimg/morlabel.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="matchingButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/gamescreen/gamescreenimg/morok.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="incomeLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Text</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:12px;
border-image: url(:/gamescreen/gamescreenimg/turuncu.png);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
